pages/api/manifest.ts

```diff
@@ -8,29 +8,24 @@ import * as GeneratedGraphQL from "../../generated/graphql";
 import { getBaseURL } from "../../lib/middlewares";
 
 const capitalize = (value: string) => value.charAt(0).toUpperCase() + value.slice(1);
+const dropFileExtension = (filename: string) => path.parse(filename).name;
 
 const inferWebhooks = async (baseURL: string) => {
   const entries = await fg(['*.ts'], { cwd: 'pages/api/webhooks' });
 
-  const dropFileExtension = (filename: string) => path.parse(filename).name;
-  const r = entries.map(dropFileExtension).map((name: string) => {
-
+  return entries.map(dropFileExtension).map((name: string) => {
     const camelcaseName = name.split('-').map(capitalize).join('');
     const statement = `${camelcaseName}SubscriptionDocument`;
-    const query = statement in  GeneratedGraphQL ? print((GeneratedGraphQL as any)[statement]) : '';
+    const query = statement in  GeneratedGraphQL ? print((GeneratedGraphQL as any)[statement]) : null;
 
     return {
       name,
       asyncEvents: [name.toUpperCase().replace("-", "_")],
       query, 
       targetUrl: `${baseURL}/api/webhooks/${name}`,
-      isActive: true,
```

Comment:
@magul are you sure? when removed, testers won't be able to test it nor specify existing handlers as `active`

Reply:
That was requested by @patrys, so maybe you will convice him that we still want it here.

Reply:
Yes, I've been thinking about something similar, i.e.

`export const isActive = true`

Reply:
All webhook handlers should either be active on installation or dropped from the manifest. When reinstalling an app, I suggest that all webhooks but the ones from the manifest are removed from Saleor before triggering the installation event. In other words, after a successful (re)installation, Saleor should have precisely the set of handlers defined in the manifest.